m4_sessions/1708-inning/group_six.md

```diff
@@ -1,9 +1,54 @@
 # Group Six
 
-### Session Title:
+### Session Title: The not so subtle art of not giving a F#^k
 
 ### Session Outcomes: 
 
+Give students the tools to determine for themselves what is important. 
```

Comment:
Determine what is important for their career?  For their life?  Or prioritizing what matters most to them?